src/Apps/Order/Components/AddressVerificationFlow.tsx

```diff
@@ -280,7 +282,11 @@ const AddressVerificationFlow: React.FC<AddressVerificationFlowProps> = ({
             <Spacer x={1} />
             <Button
               disabled={!(selectedAddressKey && selectedAddressKey.length > 0)}
-              onClick={() => {
+              onClick={e => {
+                // Because this is a type="submit" button, it will trigger the formik submit handler
```

Comment:
This button isn't `type=submit` tho; the one up above is

Reply:
I think I responded to this point above (lacking a `type` attribute, it is treated as `submit`). Why preventDefault vs stopPropagation is what formik cares about I do not know.

Reply:
My point is that the comment is incorrect; mind updating?

Reply:
```suggestion
                // Because a <button> is treated as type="submit" by default, it will trigger the formik submit handler
```